person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age, id, name = 'unknown', parent_permission: true)
+    @id = id
```

Comment:
- [OPTIONAL] Since @id will uniquely identify the a person, I think it will be better to use a random number generator, something like so: 
 ```ruby
  def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
```

Reply:
Thank you @Mihndim2020 , I will make the necessary corrections